src/Test/L0/Worker/ActionManagerL0.cs

```diff
@@ -1451,7 +1451,7 @@ public void LoadsNodeActionDefinitionYaml()
 icon: 'hello.svg' # vector art to display in the GitHub Marketplace
 color: 'green' # optional, decorates the entry in the GitHub Marketplace
 runs:
-  using: 'node12'
```

Comment:
I would not change the test code, since we have test for both node12 and node16

Reply:
Shouldn't we set up the tests for that too, then?